<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowNew</class>
 <widget class="QMainWindow" name="MainWindowNew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="viewportUpdateMode">
       <enum>QGraphicsView::MinimalViewportUpdate</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>919</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionnewFile"/>
    <addaction name="actionsaveFile"/>
    <addaction name="separator"/>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menuedit">
    <property name="title">
     <string>编辑</string>
    </property>
   </widget>
   <widget class="QMenu" name="menusetting">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actio_leftDock"/>
    <addaction name="actionProperty"/>
   </widget>
   <widget class="QMenu" name="menuhelp">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuabout">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionabout_DPSP"/>
    <addaction name="actionabout_us"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuedit"/>
   <addaction name="menusetting"/>
   <addaction name="menuhelp"/>
   <addaction name="menuabout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget_left">
   <property name="minimumSize">
    <size>
     <width>107</width>
     <height>228</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>组件窗口</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QToolBox" name="toolBox_left">
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <property name="tabSpacing">
        <number>6</number>
       </property>
       <widget class="QWidget" name="page_rader">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>256</width>
          <height>524</height>
         </rect>
        </property>
        <attribute name="icon">
         <iconset resource="../Res/res.qrc">
          <normaloff>:/img/radar.png</normaloff>:/img/radar.png</iconset>
        </attribute>
        <attribute name="label">
         <string>雷达设备</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_8">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="DragListWidget" name="listWidget_rader">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_ecm">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>256</width>
          <height>524</height>
         </rect>
        </property>
        <attribute name="icon">
         <iconset resource="../Res/res.qrc">
          <normaloff>:/img/ecmdemo.jpg</normaloff>:/img/ecmdemo.jpg</iconset>
        </attribute>
        <attribute name="label">
         <string>对抗机</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="DragListWidget" name="listWidget_ecm"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_object">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>256</width>
          <height>524</height>
         </rect>
        </property>
        <attribute name="icon">
         <iconset resource="../Res/res.qrc">
          <normaloff>:/img/input.png</normaloff>:/img/input.png</iconset>
        </attribute>
        <attribute name="label">
         <string>目标环境</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_7">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="DragListWidget" name="listWidget_object"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_property">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>属性窗口</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QListWidget" name="listWidget_property">
       <property name="autoScrollMargin">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionnewFile">
   <property name="icon">
    <iconset resource="../Res/res.qrc">
     <normaloff>:/img/new.png</normaloff>:/img/new.png</iconset>
   </property>
   <property name="text">
    <string>新建工程</string>
   </property>
   <property name="toolTip">
    <string>新建工程</string>
   </property>
  </action>
  <action name="actionsaveFile">
   <property name="icon">
    <iconset resource="../Res/res.qrc">
     <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
   </property>
   <property name="text">
    <string>保存工程</string>
   </property>
   <property name="toolTip">
    <string>保存工程</string>
   </property>
  </action>
  <action name="actionabout_DPSP">
   <property name="icon">
    <iconset resource="../Res/res.qrc">
     <normaloff>:/img/about.png</normaloff>:/img/about.png</iconset>
   </property>
   <property name="text">
    <string>关于 DPSP</string>
   </property>
  </action>
  <action name="actionabout_us">
   <property name="icon">
    <iconset resource="../Res/res.qrc">
     <normaloff>:/img/aboutUs.png</normaloff>:/img/aboutUs.png</iconset>
   </property>
   <property name="text">
    <string>关于我们</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="icon">
    <iconset resource="../Res/res.qrc">
     <normaloff>:/img/exit.png</normaloff>:/img/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actio_leftDock">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Res/res.qrc">
     <normaloff>:/img/compXmlDock.png</normaloff>:/img/compXmlDock.png</iconset>
   </property>
   <property name="text">
    <string>组件栏</string>
   </property>
  </action>
  <action name="actionProperty">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Res/res.qrc">
     <normaloff>:/img/property.png</normaloff>:/img/property.png</iconset>
   </property>
   <property name="text">
    <string>属性窗口</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DragListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">draglistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Res/res.qrc"/>
 </resources>
 <connections/>
</ui>
